draft-ietf-jsonpath-base.md

```diff
@@ -746,7 +744,7 @@ since JSON objects are unordered.
 Children of an array appear in array order in the resultant nodelist.
 
 The `wildcard` selector selects nothing from a primitive JSON value (that is,
-a number, a string, `true`, `false`, or `null`).
+a number, a string, a boolean, or `null`).
```

Comment:
I'm not sure what the comment was, but JSON defines true/false as explicit values.

It may also help us distinguish between a JSON boolean and an expression result boolean, which we've already decided are incompatible.

Reply:
The comment was a suggestion to use boolean instead of `true` and `false`. The JSON spec defines a boolean type, so this makes sense. I don't think avoiding the term "boolean" here really disambiguates the other occurrences of "boolean" since this spec should be read in the context of the JSON spec. Best if we are explicit about boolean JSON values and boolean expressions.

Reply:
Based on @cabo's comment below, I am persuaded we should stay with `true` and `false`.